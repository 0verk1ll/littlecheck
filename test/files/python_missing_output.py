# RUN: /usr/bin/python %s

import sys

print("First")
# CHECK: First

# CHECK: NotPrinted

