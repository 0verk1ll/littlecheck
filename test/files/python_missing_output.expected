Failure in python_missing_output.py:
Missing output for check:
  python_missing_output.py:8: NotPrinted
